sender.go

```diff
@@ -16,6 +16,11 @@ type Sender struct {
 	nextDeliveryTag uint64
 }
 
+// Id() is the ID of the link used for this Sender.
+func (s *Sender) Id() string {
```

Comment:
Nit: acronyms should be all upper-case.

Reply:
That's the kind of thing I'm on this team for :)